tests/unit/metadata/test_table.py

```diff
@@ -136,124 +136,92 @@ def test__get_faker_method_pass_args(self):
+        """Test that ``_make_anonymization_mappings`` method creates mappings for anonymized values.
 
-        The ``_make_anonymization_mappings`` method should create localized fake values if
-        `pii_locales` is specified in the metadata description.
+        The ``_make_anonymization_mappings`` method should map equal values in the original data
+        to the same faked value.
 
         Input:
         - DataFrame with a field that should be anonymized based on the metadata description.
-        Metadata description specifies multiple localizations using `pii_locales`.
         Output:
-        - The mappings created from the original values to localized faked values.
+        - The mappings created from the original values to faked values.
         """
         # Setup
-        metadata_dict = {
-            'fields': {
-                'foo': {
-                    'type': 'categorical',
-                    'pii': True,
-                    'pii_category': 'company',
-                    'pii_locales': ['en_US', 'sv_SE']
-                }
-            }
+        metadata = Mock()
+        metadata._ANONYMIZATION_MAPPINGS = {}
+        foo_metadata = {
+            'type': 'categorical',
+            'pii': True,
+            'pii_category': 'email'
         }
-
-        N_VALUES = 100
-        data = pd.DataFrame({'foo': list(range(N_VALUES))})
-        metadata = Table.from_dict(metadata_dict)
-
-        # Run
-        metadata._make_anonymization_mappings(data)
-        foo_mappings = metadata._ANONYMIZATION_MAPPINGS[id(metadata)]['foo']
-
-        # Assert
-        assert len(foo_mappings) == N_VALUES
-        assert list(foo_mappings.keys()) == list(range(N_VALUES))
-
-        assert 'en_US' in list(foo_mappings.values()) and 'sv_SE' in list(foo_mappings.values())
-
-    @patch('faker.generator.getattr', _mock_faker_getattr)
-    def test__make_anonymization_mappings_single_localization(self):
-        """Test that the ``_make_anonymization_mappings`` method takes single localization into account
-        when creating mappings for anonymized values.
-
-        The ``_make_anonymization_mappings`` method should create localized fake values if
-        `pii_locales` is specified in the metadata description.
-
-        Input:
-        - DataFrame with a field that should be anonymized based on the metadata description.
-        Metadata description specifies single localization using `pii_locales`.
-        Output:
-        - The mappings created from the original values to localized faked values.
-        """
-        # Setup
-        metadata_dict = {
-            'fields': {
-                'foo': {
-                    'type': 'categorical',
-                    'pii': True,
-                    'pii_category': 'company',
-                    'pii_locales': 'sv_SE'
-                }
-            }
+        metadata._fields_metadata = {
+            'foo': foo_metadata
```

Comment:
I suggest creating the other possible types of fields too, so we validate that the `if` logical checks are correct.
We should at least have:
- A field of type `id`, which should be skipped
- A field which is neither an `id` nor `pii`, which should also be skipped
- A field which is not `id` but is `pii`, for which mappings should be created

Reply:
I added checks for other types of fields in the `test__make_anonymization_mappings` test. This `test__make_anonymization_mappings_unique_faked_value_in_field ` just tests if there are repeats of a value in the id field.

Reply:
Oh! My bad, I missed this. In any case, I like how you laid it out now!